app/assets/stylesheets/users/profile/donations_by_causes.scss

```diff
@@ -0,0 +1,42 @@
+// All these are pretty much just place holders. Still playing around with them. 
+// bar sizes aren't really consistent yet. need to figure that out. 
+
+.profile-section[data-title="donations-by-causes"] {
+
+  .donations-bars {
+    display: grid;
+    grid-template-rows: 24px 16px 16px;
+    grid-gap: 3px;
```

Comment:
what is the requirement for browser support? we might have to change this if it is more strict

Reply:
browser support for CSS grid? hmm.. I don't know. I can look into that.

Reply:
yeah if for example paul says we need ie 10+ you cant use grid

Reply:
> yeah if for example paul says we need internet explorer 10+ you cant use grid

Ah I see. Then we should probably have a bigger discussion so we can all be on the same page with what browsers we need to be supporting.

Reply:
I think the better approach is to not go with the latest and greatest - you can achieve what you have here with more widely supported to not create more potential technical debt

Reply:
CSS grid is not that new.

Reply:
I mean in terms of browser support. It is definitely "new" when it comes to grid styling 

https://caniuse.com/#feat=css-grid

Reply:
I can put a question about it on trello. We probably should talk about it in the next meeting, or maybe Paul will have an answer before that.